src/gui/optionsdlg.ui

```diff
@@ -2780,8 +2780,8 @@
                   <widget class="QLineEdit" name="textWebUiAddress">
                    <property name="toolTip">
                     <string>IP address that the Web UI will bind to.
-Specify an IPv4 or IPv6 address. You can specify "0.0.0.0" for any IPv4 address,
-"::" for any IPv6 address, or "*" for both IPv4 and IPv6.</string>
+Specify an IPv4 or IPv6 address. You can specify &quot;0.0.0.0&quot; for any IPv4 address,
```

Comment:
Why plain text to HTML changes are needed?

Reply:
I noticed the change made by qt designer as well.